app/views/users/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Hello from new.html.erb</h1>
+
+<%# <form accept-charset="UTF-8" action="/users" method="post"> %>
```

Comment:
- [ ] You should be using `users_path` here 👇 
![Screen Shot 2020-11-04 at 1 03 00 AM](https://user-images.githubusercontent.com/48804258/98053706-7d5abd80-1e39-11eb-96ed-712a24684d4b.png)
```ruby
<form action="users_path" method="post" accept-charset="UTF-8">
```